client/app/helpers/optionset.js

```diff
@@ -54,40 +68,34 @@ const OPTIONSET_LOOKUP = {
  * @return     {string, number, array or Object}
  */
 export function optionset([model, optionsetId, returnType, lookupToken]) {
-  const optionset = OPTIONSET_LOOKUP[model][optionsetId];
+  const optionset = OPTIONSET_LOOKUP[model][optionsetId]; // eslint-disable-line
```

Comment:
gah, disable line? why???

Reply:
Removed. I think the was accidentally left in there. (@godfreyyeung and I had added it while improving this helper, but looks like it's not necessary now.)